Assets/Scripts/Audio/AudioManager.cs

```diff
@@ -0,0 +1,175 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Audio;
+
+public class AudioManager : MonoBehaviour
+{
+    private static AudioManager current;
```

Comment:
Just wondering, is current a conventional name for current instance? Or would instance be a more intuitive name?

Reply:
Renamed it to `instance` 👍